elastic/datadog_checks/elastic/elastic.py

```diff
@@ -802,22 +804,23 @@ def _process_metric(self, data, metric, xtype, path, xform=None,
             self._metric_not_found(metric, path)
 
     def _process_health_data(self, data, config):
-        if self.cluster_status.get(config.url) is None:
-            self.cluster_status[config.url] = data['status']
-            if data['status'] in ["yellow", "red"]:
-                event = self._create_event(data['status'], tags=config.tags)
+        status = data.get('status')
+        if not self.cluster_status.get(config.url):
+            self.cluster_status[config.url] = status
+            if status and status in ["yellow", "red"]:
```

Comment:
No need for the conjunction, if `status` is anything other than `yellow` or `red`, then this is false anyway.

Reply:
yeah, good point! Java habits die hard 😄